sqlalchemy_batch_inserts/__init__.py

```diff
@@ -153,11 +160,13 @@ def batch_populate_primary_keys(
             if skip_unsupported_models:
                 continue
             else:
-                raise AssertionError("Expected models to have auto-incrementing `id` primary key")
+                raise AssertionError("Expected models to have auto-incrementing primary key")
+
+        primary_key = _get_primary_key_name(base_mapper)
```

Comment:
Could you rename to `primary_key_name` to distinguish this pkey name (like the `_get_primary_key_name` function does) from other possible variables like pkey column (`primary_key_col`) or value?

```suggestion
        primary_key_name = _get_primary_key_name(base_mapper)
```